sdk/lib/convert/html_escape.dart

```diff
@@ -38,6 +38,23 @@ const HtmlEscape htmlEscape = HtmlEscape();
 ///
 /// Custom escape modes can be created using the [HtmlEscapeMode.HtmlEscapeMode]
 /// constructor.
+///
+/// Example:
+/// ```dart
+/// const sample = 'Text & subject, <, >, "quoted", \'single-quoted\', /';
```

Comment:
More like a question: Should it be `const String sample = 'Text & subject, <, >, "quoted", \'single-quoted\', /';` , because `.convert` takes only String value?

Reply:
Types are probably not needed in the examples

Reply:
probably not necessary but can be added.